controllers/admin.js

```diff
@@ -1,29 +1,93 @@
 const adminService = require('~/services/admin')
 
-const addAdmin = async (req, res) => {
+const inviteAdmins = async (req, res) => {
+  const admins = await adminService.inviteAdmins({
+    ...req.body,
+    language: req.lang
+  })
 
+  res.status(201).json(admins)
 }
 
-const getAdmins = async (_req, res) => {
+const getInvitations = async (req, res) => {
+  const invitations = await adminService.getInvitations()
 
+  res.status(200).json(invitations)
+}
+
+const getAdmins = async (req, res) => {
+  const {
+    skip,
+    limit,
+    name,
+    email,
+    active,
+    blocked,
+    signUpDateFrom,
+    signUpDateTo,
+    lastLoginFrom,
+    lastLoginTo,
+    sortByName,
+    sortByEmail,
+    sortByLastLogin,
+    sortBySignUpDate
+  } = req.query
```

Comment:
move query logic to a separate file in utils?

Reply:
I think we need to discuss this with Markiian